packages/wxt/src/core/wxt.ts

```diff
@@ -26,7 +26,7 @@ export async function registerWxt(
 
   const hooks = createHooks<WxtHooks>();
   const config = await resolveConfig(inlineConfig, command);
-  const builder = await createViteBuilder(config, hooks, () => wxt.server);
+  const builder = createViteBuilder(config, hooks, () => wxt.server);
```

Comment:
Hmm, and here I'm assuming vite exists lol. We should probably add an if statement and dynamic import `createViteBuilder` here as well.